stdlib/public/Platform/tgmath.swift.gyb

```diff
@@ -292,8 +292,8 @@ public func ${bfunc}(_ lhs: ${T}, _ rhs: ${T}) -> ${T} {
 %  if T == 'Float80':
 #if (arch(i386) || arch(x86_64)) && !os(Windows)
 %  else:
-//  lgamma not avaialable on Windows, apparently?
-#if !os(Windows) && !os(Android)
+//  lgamma not available on Windows, apparently?
```

Comment:
lol, not apparently, it is true, it is not.  The Windows environment does not have C99 math support.